src/measure_theory/integral/interval_integral.lean

```diff
@@ -2511,48 +2511,69 @@ end
 section smul
 
 /--
-Change of variables, general form. If `f` is continuous on `[a, b]` and has
-continuous right-derivative `f'` in `(a, b)`, and `g` is continuous on `f '' [a, b]` then we can
-substitute `u = f x` to get `∫ x in a..b, f' x • (g ∘ f) x = ∫ u in f a..f b, g u`.
-
-We could potentially slightly weaken the conditions, by not requiring that `f'` and `g` are
-continuous on the endpoints of these intervals, but in that case we need to additionally assume that
-the functions are integrable on that interval.
--/
-theorem integral_comp_smul_deriv'' {f f' : ℝ → ℝ} {g : ℝ → E}
+Change of variables, general form. If `f` is continuous on `[a, b]` and has continuous
```

Comment:
```suggestion
Change of variables, general form. If `f` is continuous on `[a, b]` and has
```
You removed this condition, right?

Other than that, LGTM

Reply:
Well spotted, I'm selling myself short :-)